src/unix/udp.c

```diff
@@ -349,7 +349,11 @@ int uv__udp_bind(uv_udp_t* handle,
 static int uv__udp_maybe_deferred_bind(uv_udp_t* handle,
                                        int domain,
                                        unsigned int flags) {
-  unsigned char taddr[sizeof(struct sockaddr_in6)];
+  union {
+    struct sockaddr_in6 in6;
+    struct sockaddr_in in;
+    struct sockaddr addr;
+  } taddr;
```

Comment:
`sockaddr_storage` is the largest of any `sockaddr` derivative, possibly much larger than either `sockaddr_in` or `sockaddr_in6`.